src/unitxt/operators.py

```diff
@@ -1028,27 +1028,70 @@ def process(
 
 
 class FilterByValues(SingleStreamOperator):
-    """Filters a stream, yielding only instances that match specified values in the provided fields.
+    """Filters a stream, yielding only instances that match specified required values in the provided fields and do not match specified disallowed values.
+
+    Raises an error if a required key is missing.
 
     Args:
-        values (Dict[str, Any]): For each field, the values that instances should match to be included in the output.
+        required_values (Dict[str, Any]): Values that instances must match to be included in the output.
+        disallowed_values (Dict[str, Any]): Values that instances must not match to be excluded from the output.
```

Comment:
must not match to be **included in** the output

Perhaps emphasize with stars, in both lines *must match*  and *must not match*